### ISSUE TITLE
Operation off scale message and text box behave oddly with long strings

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/number-line-operations#87]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/597.
When strings are long, the text box that appears that says that an operation is off the scale appears much longer than it needs to be.
**Steps to reproduce**
1. Go to the Operations screen with `?stringTest=double` or `?stringTest=long`
2. Add an operation
3. Drag the starting point to the far right (or left) of the numberline, so that you get the Operation Off Scale message

**Visuals**
<img width="357" alt="longoffmessage" src="https://user-images.githubusercontent.com/41024075/105249178-ed253e00-5b34-11eb-9b49-51434e4a61bf.png">
<img width="219" alt="oddmessage" src="https://user-images.githubusercontent.com/41024075/105249182-ee566b00-5b34-11eb-82e5-61aa29ef749a.png">

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Number Line: Operations‬:‪Number Line: Operations‬
URL: https://phet-dev.colorado.edu/html/number-line-operations/1.0.0-dev.13/phet/number-line-operations_all_phet.html?stringTest=double
Version: 1.0.0-dev.13 2021-01-16 00:14:30 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.141 Safari/537.36
Language: en-US
Window: 1280x658
Pixel Ratio: 1.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jbphet** [comment]
This was due to a problem with `BackgroundNode` in the scenery-phet repo.  The background was being scaled on changes to the local bounds, but local bounds aren't changed when a node is scaled down due to a `maxWidth` parameter.  I changed this to use the overall bounds instead and, at @pixelzoom's suggestion, added an intermediate node to avoid potentially problematic recursion when positioning the content node.

I'll leave this open and will have it verified in the next release.
